app/src/main/java/fr/iut_rodez/pathpilot_android_client/util/Network.java

```diff
@@ -0,0 +1,65 @@
+package fr.iut_rodez.pathpilot_android_client.util;
+
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
+import com.android.volley.RequestQueue;
+import com.android.volley.toolbox.Volley;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Network {
```

Comment:
Missing Javadoc comment to describe what the class is for

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/38/commits/eb823270691dc0c912b801466992d2ce62dfca4e